packages/compiler-sfc/__tests__/compileScriptPropsTransform.spec.ts

```diff
@@ -99,6 +99,33 @@ describe('sfc props transform', () => {
     assertCode(content)
   })
 
+  test('default values w/ type declaration & key is string', () => {
+    const { content, bindings } = compile(`
+      <script setup lang="ts">
+      const { foo = 1, bar = 2 } = defineProps<{ 
+          "foo": number // double-quoted string
+          "bar": number // single-quoted string
+          'foo:bar': string // single-quoted string containing symbols
+          "onUpdate:modelValue": (val: number)=>void } // double-quoted string containing symbols
+          >()
+      </script>
+    `)
+    expect(bindings).toStrictEqual({
+      foo: BindingTypes.PROPS,
+      bar: BindingTypes.PROPS,
+      "'foo:bar'": BindingTypes.PROPS,
+      "'onUpdate:modelValue'": BindingTypes.PROPS
```

Comment:
I think quotes should not be included in `bindings`.

Reply:
Yes, I'm still working on it, and there seems to be a problem with the default value of the alias